Source/WebCore/rendering/svg/SVGTextLayoutEngine.cpp

```diff
@@ -249,23 +249,23 @@ void SVGTextLayoutEngine::layoutInlineTextBox(SVGInlineTextBox& textBox)
 #if DUMP_TEXT_FRAGMENTS > 0
```

Comment:
This seems a very generic name, maybe we can consider making the name clear so it deals with SVG text engine.

Reply:
Ok.

Reply:
DUMP_SVG_TEXT_LAYOUT_FRAGMENTS?